assets/css/edd-admin.css

```diff
@@ -2431,6 +2414,57 @@ body.download_page_edd-reports {
 	margin-bottom: 20px;
 }
 
+div.edd-reports-export {
+	display: grid;
+	display: -ms-grid;
+	grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+	grid-gap: 20px;
+}
+
+.edd-reports-wrapper .postbox {
+	margin-bottom: 0;
+}
+
+.edd-export-sales-earnings {
+	grid-template-columns: 1fr;
+}
+
+[class^="edd-export"] .edd-from-to-wrapper {
+	display: flex;
+	margin-right: 0;
+	margin-bottom: 16px;
+	width: 100%;
+}
+
+.download_page_edd-reports .inside input[type="text"] {
+	width: 100%;
+}
+
+.edd-import-export-form {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+}
+
+div.edd-to-and-from-container {
+	display: grid;
+	display: -ms-grid;
+    grid-template-columns: 1fr 1fr;
+	grid-gap: 8px;
+}
+
+span.edd-label.edd-export-to-label {
+	line-height: normal;
+	align-self: center;
+	margin-bottom: 16px;
+
+}
+
+.edd-reports-wrapper .postbox .edd-select {
```

Comment:
I think this is intended to override the existing ruleset defined beginning at line 2355, which applies to all `select` elements on reports, so this should not use the `.edd-reports-wrapper` alone--needs `.edd-reports-export` instead or as well. (Also likely that the margin-bottom in the above ruleset should be moved to here, although I'm not sure it's negatively impacting other reports screens that I can see so far.)